cl_sii/dte/data_models.py

```diff
@@ -495,32 +664,32 @@ class DteXmlData(DteDataL1):
     # fields
     ###########################################################################
 
-    emisor_razon_social: str = dc_field()
+    emisor_razon_social: str
```

Comment:
I would move this, and similar changes, into a separate commit and/or pull request.